Content/build.fsx

```diff
@@ -19,6 +19,10 @@ open Cit.Helpers.Arm
 open Cit.Helpers.Arm.Parameters
 open Microsoft.Azure.Management.ResourceManager.Fluent.Core
 //#endif
+//#if (deploy == "gcp-kubernetes")
+open System.Text.RegularExpressions
+open Fake.Tools.Git
```

Comment:
Just thinking - is it really crucial to employ Git here? Can we find a different way to create a docker tag? Adding this dependency has 2 main drawbacks:
* it implies the user needs to actually create git repo and commit before testing out the deployment
* makes template a bit harder to maintain

Reply:
It can be solved with a version variable. The reason I added it is because I see it as a good example of how to version your docker container based on your git sha, which does make some things around versioning slightly simpler in some scenarios, and also deterministic.

If it adds a lot of complexity I can, of course, figure something else out. What do you think based on motivation to why I added it in the first place @theimowski?

Reply:
Yes I totally agree it's a go-to solution for deterministic tagging.
However in case of a template it's more steps for a user to try out functionality, and we strive to make the template as easy to use as possible and with minimal steps to deploy.

Can you push to Google Container Registry a docker image without tag (`projectId/projectName` instead of `projectId/projectName:tag`)? 
If so maybe this could be the easiest way to start with. Otherwise - what would a "version variable" look like?